lib/dealership.rb

```diff
@@ -0,0 +1,44 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :inventory_count
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
```

Comment:
Nice job here. Another way that you could do `inventory_count` is to create a method and call
```
@inventory.length
```